### ISSUE TITLE
Distance bug ?[BUG]

### ISSUE THREAD

**Issue by ErolSaydam** [hlorus/CAD_Sketcher#33]
**Description**
Start blender 3.1.2 , then delete all defaults, push N, add sketch, then add rectangle , then make construction line as seen in the screenshot
then click middelpoint (didnt get orange) the click construction line, then click constraints menu choose Distance, then click (see screenshot) upper right and then getting the error message

**To Reproduce**
1.  See above
2.  tried to reproduce and I getting error message again

**Expected behavior**
I was expecting to click the first point then the second and filling in a distance for this partical line and learning from the youtube video

**Screenshots**
yes 2 screenshots

![bug blender 3 1 2](https://user-images.githubusercontent.com/40236070/164951508-b201890e-0ebd-4149-acf1-4d9b33ead823.png)
![bug blender 3 1 2_2](https://user-images.githubusercontent.com/40236070/164951511-8628d86b-a2c0-41bd-b9c4-834df3742033.png)

trying reproduce and yes, this message:

![image](https://user-images.githubusercontent.com/40236070/164951607-c729e136-7105-40c8-9322-fb3fd0aadd70.png)

**System Information:**
 - OS:   Windows 10 pro english
 - Blender Version:  3.1.2
 - Addon Version: 0.20.0 Experimental

**Comment by teddywaweru** [comment]
I think the normal workflow was that you select the entities that you want to set the constraints on first ie. select the midpoint, then the other point, then click on the Distance constraint.
I could replicate the above error by:
1. clicking first point.
2. click distance Constraint
3. click second point. 

Meanwhile the intended workflow is:
1. click first point
2. click second point
3. click Distance constraint.

Basically the Constraint click always expects that two points are currently highlighted prior. It would be necessary to have the program report the error appropriately so that users understand that no points had been selected in the operation.
